editor/inspector/editor_resource_picker.cpp

```diff
@@ -1076,7 +1076,7 @@ void EditorResourcePicker::_notification(int p_what) {
 			draw_style_box(get_theme_stylebox(SceneStringName(panel), SNAME("Tree")), Rect2(Point2(), get_size()));
 		} break;
 
-		case NOTIFICATION_RESIZED: {
```

Comment:
This case was _added_ in #109458
I wonder how it was working before 🤔

Reply:
The bug itself is caused by `preview_rect` having a width of `0` by the time it's called, which is lower than `thumbnail_size`, triggering the hide behavior. Said code happens inside `_update_resource_preview()`.

As to how it was working before? I'm honestly not quite sure. :shrug: